android/capacitor/src/main/java/com/getcapacitor/plugin/camera/ImageUtils.java

```diff
@@ -17,6 +17,21 @@
 
 public class ImageUtils {
 
+  /**
+   * Resize an image to the given width and height considering the preserveAspectRatio flag.
+   * @param bitmap
+   * @param width
+   * @param height
+   * @param preserveAspectRatio
+   * @return a new, scaled Bitmap
+   */
+  public static Bitmap resize(Bitmap bitmap, final int width, final int height, final boolean preserveAspectRatio) {
```

Comment:
Will this be a breaking change? You've added a new parameter to this public method in the Android platform. Will calls such as `ImageUtils.resize(bitmap, 100, 100)` continue to work?

You can mitigate this by adding the following method:

```java
  public static Bitmap resize(Bitmap bitmap, final int width, final int height) {
    resize(bitmap, width, height, false);
  }
```

Reply:
Actually I think it's not. There's only one call of this static method and it's [here](https://github.com/ionic-team/capacitor/pull/3309/files/79863659188fcd59bd219a004f37e2226abf0bcb#diff-436e54d5c31c5e7863a079db6374c887R420-R425). But I've added an overload anyway. 👍🏽